src/wallet/api/wallet.cpp

```diff
@@ -524,12 +534,12 @@ bool WalletImpl::recoverFromKeys(const std::string &path,
     try
     {
         if (has_spendkey) {
-            m_wallet->generate(path, "", info.address, spendkey, viewkey);
+            m_wallet->generate(path, password, address, spendkey, viewkey);
```

Comment:
Why did `address` change to `info.address`? Is `address` even being defined somewhere?

Reply:
nice catch. compiler didn't complain. and recovery seemed to work. fixed.

Reply:
hah - figured it out. master needs "info." v0.11.1.0 does'nt.